sql/hive/src/main/scala/org/apache/spark/sql/hive/execution/SaveAsHiveFile.scala

```diff
@@ -68,6 +68,26 @@ private[hive] trait SaveAsHiveFile extends DataWritingCommand {
         .get("mapreduce.output.fileoutputformat.compress.type"))
     }
 
+    fileSinkConf.tableInfo.getOutputFileFormatClassName match {
+      case formatName if formatName.endsWith("ParquetOutputFormat") =>
+        val compressionConf = "parquet.compression"
+        val compressionCodec = getCompressionByPriority(fileSinkConf, compressionConf,
+          sparkSession.sessionState.conf.parquetCompressionCodec) match {
```

Comment:
```Scala
 val compressionCodec = getCompressionByPriority(
    fileSinkConf, 
    compressionConf = "parquet.compression",
    default = sparkSession.sessionState.conf.parquetCompressionCodec) match {
```

Reply:
`compressionConf` will be used below, I've adjusted the format, thanks.